src/base.js

```diff
@@ -13,11 +13,17 @@ class Base extends EventEmitter {
     this.bodyAttributes = [];
   }
 
-  addSoapHeader(value, qname, options) {
-    var header = new SOAPElement(value, qname, options);
+  addSoapHeader(name, value, qname, options) {
```

Comment:
qname existed already in soapHeader. name is added because, in client.js  code was referring to soapHeader.name. I guess, we should derive the name from the qname. I will go ahead and make the changes.